readme.md

```diff
@@ -27,11 +27,21 @@ See the "require" section of [composer.json](https://github.com/silverstripe/sil
 ## Installation
 
 ```sh
-$ composer require silverstripe/userforms
+composer require silverstripe/userforms
 ```
 
 You'll also need to run `dev/build`. You should see a new page type in the CMS 'User Defined Form'. This has a new 'Form' tab which has your form builder.
 
+## SPAM protection
+
+This module does not include SPAM protection out of the box. Consider installing `silverstripe/spamprotection` as well. This will allow CMS users to add a "SPAM Protection Field" to user forms.
```

Comment:
Need to note in this readme that `silverstripe/spamprotection` is just an abstraction, you also need to add another module on top of that.  As mentioned in the parent issue, installing this module without the implementation will give a false sense of security as users can add a spam protection field that does nothing. There are some common modules list in the readme of `silverstripe/spamprotection`

Most commonly used one I've seen by far is the undefinedoffset/nocaptcha (google recaptcha), though as mentioned on the parent issue of this PR, we don't officially support that module, and what's most popular may change, so we can't really recommend use of that module within this readme.